governance/SIG Network Charter.md

```diff
@@ -1,97 +1,109 @@
 
 ## In scope
 - Define standard for communication negotiation.
-- Define and maintain implementation to enable compression and secure communication between endpoints
-- Define and implement interface to network counters, monitoring, and statistics
-- Define and implement low level instrumentation APIs
+- Define and maintain implementation to enable compression and secure communication between endpoints.
+- Define and implement interface to network counters, monitoring, and statistics.
+- Define and implement low level instrumentation APIs.
+- Items that are the core responsibilities of SIG.
+- Maintenance of network sample projects owned by SIG: [MultiplayerSample](https://github.com/o3de/o3de-multiplayersample), [NetSoakTest](https://github.com/o3de/o3de-netsoaktest).
 
 ###Network:
-- Design and maintain client and server protocol communication 
+- Design and maintain client and server protocol communication.
 - Define, publish, and maintain network protocol and data packet design model.
 - Responsible for network presence and state replication.
-- Create and Maintain network simulation tools for testing network condititons.
-
-- Design and maintain multiplayer component architecture supporting client hosted and dedicated server strategies
+- Create and Maintain network simulation tools for testing network conditions.
+- Design and maintain multiplayer component architecture supporting client hosted and dedicated server strategies.
 - Define and maintain specific implementation of multiplayer controller component.
+- Networking specific serialization (as part of the transport layer).
 
 ###Cloud:
-- Design and implement network communication and data exchange for cloud services.
-- Define, standardize, and maintain interface to feature based cloud services.
-- Define and maintain code to enable cloud data services communication
-- Define and implement 3rd party distribution platform services interfaces.
-
-for identity, matchmaking, profiles, persistence.
-
-- Items that are the core responsibilities of SIG
-
-**Cross-cutting Processes**
-- consume and utilize services
-- ensure client and server targets
-- Define communication protocol and stack
-- Define and maintain entity replication network model
-- Responsible for abstraction of console design for dedicated / headless server implementation (core / init)
+- Define, standardize, and maintain interfaces for features based on cloud services.
+- Define and maintain code to enable cloud data services communication.
+
+###Cross-cutting Processes
+- Define client and server build targets for supported platforms, cross-cutting issue with `#sig-core` and `#sig-build`.
+- Define communication protocol and stacks.
+- Define and maintain entity replication network model.
+- Responsible for abstraction of console design for dedicated / headless server implementation (core / init).
```

Comment:
what's core/init? the base-classes and patterns used for the core functionality and how systems are initialized?